sample_code/ep451/README.md

```diff
@@ -0,0 +1,29 @@
+# [why I banned python's assertRaises (intermediate)](https://youtu.be/Q_HMOBD09ts)
+
+Today I walk through why I banned `assertRaises` in tests at work as well as an alternative that doesn't have this pitfall
+
+## Interactive examples
+
+### Python debugger (pdb)
+
+```
+x
+y
+up
+x
+y
+q
+```
+
+### Bash
+
+```bash
+python -m unittest t
```

Comment:
Test discovery for you didn't work, because the name of the file was `t.py` and by default unittest searches for `test*.py` patterns :D ([unittest Test Discovery](https://docs.python.org/3/library/unittest.html#cmdoption-unittest-discover-p))

Reply:
ugh oops